kratos/geometries/quadrilateral_3d_4.h

```diff
@@ -515,7 +515,7 @@ template<class TPointType> class Quadrilateral3D4
         const double Tolerance = std::numeric_limits<double>::epsilon() 
         ) override
     {
-        PrivatePointLocalCoordinates( rResult, rPoint, true );
+        PointLocalCoordinatesImplementation( rResult, rPoint, true );

```

Comment:
Why did you change the name of the method? IMO PrivatePointLocalCoordinates is more self explanatory.

Reply:
I did the change @pooyan-dadvand  suggested

Reply:
We have used the implementation suffix many places when it is a function body to be used in different methods. The private is already marked by the type of the method (private/public). For classes the new `Internals `namespace (or nested class) should be used